### PR TITLE
[ACC-339] Add route to restart server

### PR DESCRIPTION
**Pull request by gabrielseibel1**

### CODE REVIEW

File: controllers/server.go [hunk 1]
```diff
@@ -18,16 +19,24 @@ func serverHandlers(r *mux.Router) {
...
 			resp.WriteHeader(http.StatusOK)
 			resp.Write([]byte("Server is up and running!!"))
-		}),
-	)
+		},
+	).Methods(http.MethodGet)
+	r.HandleFunc(
+		"/api/server/health",
+		func(w http.ResponseWriter, _ *http.Request) {
+			_ = syscall.Kill(syscall.Getpid(), syscall.SIGINT)
```
Comment (Aceix) [inline controllers/server.go:30]: can we add a log before the sigkill

**Review by Aceix — CHANGES_REQUESTED** [review]

File: controllers/server.go [hunk 2]
```diff
@@ -18,16 +19,24 @@ func serverHandlers(r *mux.Router) {
...
-		http.HandlerFunc(func(resp http.ResponseWriter, req *http.Request) {
+		func(resp http.ResponseWriter, req *http.Request) {
 			resp.WriteHeader(http.StatusOK)
 			resp.Write([]byte("Server is up and running!!"))
-		}),
-	)
+		},
+	).Methods(http.MethodGet)
+	r.HandleFunc(
+		"/api/server/health",
```
Comment (abhishek9686) [inline controllers/server.go:28]: change api route

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: controllers/server.go (hunk 2 above)
Comment (gabrielseibel1) [reply controllers/server.go:28]: any suggestions? i found acceptable to have GET server/health and DELETE server/health to check if its on or turn it off haha

File: controllers/server.go (hunk 1 above)
Comment (gabrielseibel1) [reply controllers/server.go:30]: good idea

File: controllers/server.go (hunk 2 above)
Comment (gabrielseibel1) [reply controllers/server.go]: https://github.com/gravitl/netmaker/pull/2672/commits/b1ece5f72e3aae7bb4ecb5a9885bd8b87785150d

File: controllers/server.go (hunk 1 above)
Comment (gabrielseibel1) [reply controllers/server.go]: https://github.com/gravitl/netmaker/pull/2672/commits/59add9dd04d2923ac7757467867c9d1e77fc6ca0

**Review by abhishek9686 — APPROVED** [review]

**Pull request merged**